src/main/controllers/ContactTheTribunalSelectedController.ts

```diff
@@ -100,13 +100,13 @@ export default class ContactTheTribunalSelectedController {
     req.session.errors = [];
 
     userCase.contactApplicationSending = true;
-    await submitClaimantTse(req, logger);
-    clearTseFields(userCase);
```

Comment:
We want to keep TSE fields in the session as we go through the journey so removed this. ClearTSE is in the get for this page so that handles the clearing for starting a new application.

Reply:
We want to clear the tse values if we submit an application becasue then if we go to the same type of application to make a new one, those values will still appear there. Might be a problem, I don't know